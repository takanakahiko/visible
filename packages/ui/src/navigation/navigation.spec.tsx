import React from 'react';
import { render } from '@testing-library/react';
import { Nav, NavItem } from '.';

describe('Navigation', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <Nav>
        <NavItem>test</NavItem>
      </Nav>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
