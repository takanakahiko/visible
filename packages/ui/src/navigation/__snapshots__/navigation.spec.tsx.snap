// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0;
  padding: 0;
  list-style: none;
}

.c3 {
  width: 100%;
  height: 1.5px;
  background-color: #ffaa01;
}

.c1 {
  display: block;
  margin: 0 18px;
}

.c1:last-child {
  margin-right: 0;
}

.c1 > a {
  display: block;
  box-sizing: border-box;
  padding: 12px 0px;
  -webkit-transition: 0.15s ease-out;
  transition: 0.15s ease-out;
  color: #333333;
}

.c1 > a:hover {
  color: #ffaa01;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c1 .c2 {
  display: none;
}

<ul
  class="c0"
>
  <li
    class="c1"
  >
    test
    <div
      class="c2 c3"
    />
  </li>
</ul>
`;
